extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -131,4 +140,39 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
       }
     }
   }
+
+  def getOrCreateRangerPlugin(catalog: Option[String] = None): RangerBasePlugin = synchronized {
+    catalog match {
+      case None | Some("spark_catalog") =>
+        defaultBasePlugin
+      case Some(catalogName) =>
+        val serviceName = getRangerConf(defaultBasePlugin)
+          .get(s"ranger.plugin.spark.catalog.$catalogName.service.name")
+        serviceName match {
+          case _ if StringUtils.isBlank(serviceName) =>
+            logWarning(s"config ranger.plugin.spark.catalog.$catalogName.service.name not found," +
+              s" default ranger plugin is used")
+            defaultBasePlugin
```

Comment:
Here returns a `defaultBasePlugin` for the catalog without setting the Ranger service name.
But it may shadow and block the normal access to the catalog.
How to return a allow-all ranger plugin to skip access checking for them?